templates/php/src/Client.php.twig

```diff
@@ -14,6 +14,8 @@ class Client
     const METHOD_CONNECT = 'CONNECT';
     const METHOD_TRACE = 'TRACE';
 
+    const CHUNK_SIZE=5000000;
```

Comment:
Can we have delimitier between 3 char or a comment.
```suggestion
    const CHUNK_SIZE=5_000_000;
```

Reply:
@TorstenDittmann seems to cause problem in php 7
![image](https://user-images.githubusercontent.com/6360216/127445311-ed4c744b-7afa-4980-b17c-19c3febaa6ba.png)